### PR TITLE
QSA-22-18

### PR DESCRIPTION
**Pull request by stanleyshuang**
CVE-2021-34360

### CODE REVIEW

File: 2021/34xxx/CVE-2021-34360.json [hunk 1]
```diff
@@ -1,18 +1,137 @@
 {
```
Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'references' is a required property

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34360.json:1]: 'description' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hello @stanleyshuang ,

Please resolve the merge conflict. Once resolved the automated process will continue.

Thanks,
CVE Content Team

**Comment by anthnysingleton** [comment]
Hello @stanleyshuang,

The CVE Github Pilot does not accept JSON v5. You can only submit JSON v4 through this endpoint. 

Closing PR.

Thanks,
CVE Content Team

**Pull request closed without merging**